packages/elements-react/src/components/form/form.tsx

```diff
@@ -37,46 +37,147 @@ import {
-  Label: ComponentType<HeadlessLabelProps>
-  Checkbox: ComponentType<HeadlessInputProps>
-  Text: ComponentType<HeadlessTextProps>
-
-  FormContainer: ComponentType<HeadlessFormProps>
-
-  SocialButton: ComponentType<HeadlessSocialButtonProps>
-  SocialButtonContainer: ComponentType<HeadlessSocialButtonContainerProps>
-
-  AuthMethodListItem: ComponentType<HeadlessAuthMethodListItemProps>
+/**
+ * A record of all the components that are used in the OryForm component.
+ */
+export type OryFlowComponents = {
+  Node: {
+    /**
+     * Button component, rendered whenever a button is encountered in the Ory UI Nodes.
+     */
+    Button: ComponentType<OryNodeButtonProps>
+    /**
+     * The SocialButton component is rendered whenever a button of group "oidc" node is encountered.
+     *
+     * It renders the "Login with Google", "Login with Facebook" etc. buttons.
+     */
+    OIDCButton: ComponentType<OryNodeOIDCButtonProps>
+    /**
+     * The CurrentIdentifierButton component is rendered whenever a button of group "identifier_first" node is encountered.
+     *
+     * It is used to show the current identifier and can allow the user to start a new flow, if they want to.
+     */
+    CurrentIdentifierButton: ComponentType<OryCurrentIdentifierProps>
+    /**
+     * Anchor component, rendered whenever an "anchor" node is encountered
+     */
+    Anchor: ComponentType<OryNodeAnchorProps>
+    /**
+     * The Input component is rendered whenever a "input" node is encountered.
+     */
+    Input: ComponentType<OryNodeInputProps>
+    /**
+     * Special version of the Input component for OTP codes.
+     */
+    CodeInput: ComponentType<OryNodeInputProps>
+    /**
+     * The Image component is rendered whenever an "image" node is encountered.
+     *
+     * For example used in the "Logo" node.
+     */
+    Image: ComponentType<OryNodeImageProps>
+    /**
+     * The Label component is rendered around Input components and is used to render form labels.
+     */
+    Label: ComponentType<OryNodeLabelProps>
+    /**
+     * The Checkbox component is rendered whenever an input node with of boolean type is encountered.
+     */
+    Checkbox: ComponentType<OryNodeInputProps>
+    /**
+     * The Text component is rendered whenever a "text" node is encountered.
+     */
+    Text: ComponentType<OryNodeTextProps>
+  }
+  Card: {
+    /**
+     * The card container is the main container of the card.
+     */
+    Root: ComponentType<OryCardRootProps>
+    /**
+     * The card footer is the footer of the card container.
+     */
+    Footer: ComponentType<OryCardFooterProps>
+    /**
+     * The card header is the header of the card container.
+     */
+    Header: ComponentType<OryCardRootProps>
+    /**
+     * The card content is the main content of the card container.
+     */
+    Content: ComponentType<OryCardContentProps>
+    /**
+     * The card logo is the logo of the card container.
+     */
+    Logo: ComponentType
```

Comment:
Missing props?

Reply:
I think it doesn't need any props. But for consistency, maybe?

Reply:
Done.